CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Stack overflow when printing shader creation error
 - Underline position for bitmap fonts
 - Selection rotating outside of scrolling region
+- Minor throughput performance problems due to excessive font metric queries
```

Comment:
Let's just say that it doesn't do anything for me...

Reply:
Shouldn't matter, as long as it does something for some people.